src/main/webapp/firebase-chat.js

```diff
@@ -0,0 +1,280 @@
+/* eslint-disable */
+const firebaseConfig = {
```

Comment:
Can you moving this config to firebase_config.js? 

These details can be used for other usecases too right?